### PR TITLE
[W6.1h][T09-B3] Niu Yunpeng

### PR DESCRIPTION
**Pull request by yunpengn**
- Add Printable Interface
- Let phone, email, address and name _implement_ Printable
- Add test case

### CODE REVIEW

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -43,10 +44,23 @@ public String format(List<? extends ReadOnlyPerson> persons) {
         return format(asIndexedList(formattedPersons));
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public String getPrintableString(Printable... printables){
+        StringBuilder sb = new StringBuilder();
```
Comment (ngsikai) [inline src/seedu/addressbook/ui/Formatter.java:51]: Good job using the StringBuilder class, another alternative is the StringJoiner Class.

Reference: https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html

**Comment by ngsikai** [comment]
Go ahead and close the PR.

**Comment by ngsikai** [comment]
Reopening PR as I accidentally closed it.

File: src/seedu/addressbook/ui/Formatter.java (hunk 1 above)
Comment (yunpengn) [reply src/seedu/addressbook/ui/Formatter.java:51]: Noted with thanks.

**Comment by yunpengn** [comment]
Close this PR since it has been reviewed by the tutor.

**Pull request closed without merging**